gnirehtet.cmd

```diff
@@ -0,0 +1,149 @@
+@ECHO OFF
+CALL :locate_file ADB "%LOCALAPPDATA%\Android\sdk\platform-tools\adb.exe"
+CALL :locate_file JAVA "%JAVA_HOME%\bin\java.exe"
+CALL :locate_file APK %~dp0gnirehtet.apk %~dp0app\build\outputs\apk\gnirehtet-release.apk %~dp0app\build\outputs\apk\gnirehtet-debug.apk
+CALL :locate_file RELAY %~dp0relay.jar %~dp0relay\build\libs\relay.jar
+
+::main
+IF NOT "%2"=="" SET serial="-s %2"
+CALL :do_%1 %serial% 2>NUL || CALL :do_help 
+GOTO :eof
+
+:do_help
+    ECHO Syntax: %~nx0 (install^|uninstall^|rt^|start^|stop^|relay^|killserver^|kill) ...
+    ECHO.
+    ECHO  %~nx0 install [serial]
+    ECHO      Install the client on the Android device and exit.
+    ECHO      If several devices are connected via adb, then serial must be
+    ECHO      specified.
+    ECHO.
+    ECHO  %~nx0 uninstall [serial]
+    ECHO      Uninstall the client from the Android device and exit.
+    ECHO      If several devices are connected via adb, then serial must be
+    ECHO      specified.
+    ECHO.         
+    ECHO  %~nx0 reinstall [serial]
+    ECHO      Uninstall then install.
+    ECHO.
+    ECHO  %~nx0 rt [serial] [DNS[,DNS2,...]]
+    ECHO      Enable reverse tethering for exactly one device:
+    ECHO        - install the client if necessary;"
+    ECHO        - start the client;
+    ECHO        - start the relay server;
+    ECHO        - You'll have to stop the client and the server manually
```

Comment:
Since it is not managed by `gnirehtet rt`, it should not be a list item IMO:

     - start the client;
     - start the relay server.
    You'll have to stop the client and the server manually.

`</nitpicking>`